apps/frontend/src/app/legal/legal/legal.component.html

```diff
@@ -89,5 +89,19 @@ <h3>User Rights</h3>
       </p>
     </li>
   </ul>
-  <h3>Contact Us</h3>
+  <div *ngIf="imprint">
+    <h3>Contact Us</h3>
+    <p>
+      {{imprint.contactOperator}}<br>
+      {{imprint.contactAddress}}<br>
+      <a href="mailto:{{imprint.contactMail}}">{{imprint.contactMail}}</a>
+    </p>
+  </div>
+  <div *ngIf="!imprint">
```

Comment:
I would just not show this section

Reply:
I removed it in [c7b2028](https://github.com/Morphclue/apollusia/pull/95/commits/c7b2028be722ff9d150b0658f5d86d90ba6dff87).